android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/sessions/SpecialOffersActivity.java

```diff
@@ -34,22 +38,17 @@ public void error(Exception e, String userAction) {
 		int backgroundColor =
 			ContextCompat.getColor(this, com.liferay.mobile.screens.viewsets.lexicon.R.color.lightRed);
 
-		AndroidUtil.showCustomSnackbar(webContentDisplayScreenlet, userAction, Snackbar.LENGTH_LONG, backgroundColor,
-			Color.WHITE, icon);
+		AndroidUtil.showCustomSnackbar(webScreenlet, userAction, Snackbar.LENGTH_LONG, backgroundColor, Color.WHITE,
+			icon);
 	}
 
 	@Override
-	public WebContent onWebContentReceived(WebContent html) {
-		return null;
-	}
+	public void onPageLoaded(String url) {
```

Comment:
Hey @mclomello please add a while loading the screenlet page and inside this method you can hide it.